---
layout: default
title: "CV of Xin-Qiu Yao"
---
[Download CV as PDF](CV_XQYao.pdf).

# Xin-Qiu YAO, Ph.D.

2055 Palmer Commons <br>
100 Washtenaw Avenue <br>
Ann Arbor, MI 48109-2218, U.S.A. <br>
Tel: +1(734)-615-6727 <br>
E-Mail: <xinqyao@umich.edu> <br>
Website: <https://xinqyao.github.io> <br>
<br>

## PROFESSIONAL POSITIONS
<p> <em>09. 2012 – present</em>, Research Fellow (Barry Grant’s Lab), Dept Computational Medicine & Bioinformatics, University of Michigan </p>
<p> <em>09. 2008 – 07. 2012</em>, Research Fellow (Shoji Takada’s Lab), Dept Biophysics, Grad School of Science, Kyoto University </p>
---
 
## EDUCATION
<p> <em>09. 2005 – 06. 2008</em>, <strong> Ph.D.</strong>, Biomechanics and Medical Engineering, College of Engineering, Peking University (Advisors: Zhen-Su She & Huaiqiu Zhu. Title of dissertation: “A complex systems approach to study of protein structures”) </p>
<p> <em>09. 2002 – 06. 2005</em>, Graduate student, Condensed Matter Physics, School of Physics, Peking University </p>
<p> <em>09. 1998 – 06. 2002</em>, <strong>B.S.</strong>, Applied Physics (Minored in Japanese), Dept Applied Physics, Hunan University </p>
---

## INTERNSHIP AND OTHER TRAINING
<p> <em>07. 2002 – 07. 2008</em>, <strong>RA</strong>, Center for Theoretical Biology (Zhen-Su She’s Lab), Peking University, Beijing, China </p>
<p> <em>07. 2003</em>, Trainee, School of Parallel Computing, Chinese Academy of Science, Beijing, China </p>
<p> <em>10. 2004 – 12. 2004</em>, <strong>Internship</strong>, Intel China Research Center, Beijing, China </p>
---

## TEACHING AND MENTORING
<p> <em>2003 (Fall)</em>, <strong>TA</strong>, Statistical Physics, School of Physics, Peking University </p>
---

## GRANT APPLICATION
<p> <em>2015</em>, <strong>Co-I</strong>, “G Protein Activation Mechanisms” (Proposal for allocation of simulation time on Anton at the Pittsburgh Supercomputing Center, Grant No. MCB150033P) <strong>[Awarded]</strong> </p>
---

## HORNORS AND AWARDS
<p> <em>2011</em>, <strong>The Best Talk</strong>, “The 13th Annual Meeting of Chinese Life Science Association”, Kyoto, Japan </p>
<p> <em>2010</em>, <strong>The Best Poster</strong>, “The 2nd Biosupercomputing Symposium”, Tokyo, Japan </p>
<p> <em>2008</em>, General Presentation Award, “Bio-rad Forum of Graduate Students”, Peking University, Beijing, China </p>
<p> <em>2000</em>, General Scholarship, Hunan University, Changsha, China </p> 
<p> <em>1999</em>, General Scholarship, Hunan University, Changsha, China </p>
---

## AFFILLIATIONS AND SOCIAL SERVICES

*Academic*
<p> <em>2009 – present</em>, Membership, Biophysical Society </p>
<p> <em>2010 – 2012</em>, Membership, Biophysical Society of Japan </p>

*Non-academic*
<p> <em>2009 – 2012</em>, General Secretary, Chinese Scholars Association in Kyoto </p>
<p> <em>1999 – 2002</em>, Student Committee, Dept. Applied Physics, Hunan University </p>
<p> <em>1998 – 1999</em>, President Assistant, Philately Society, Hunan University </p>
---

## SKILLS
OS: Linux, Windows, Mac OS X <br>
Software: Amber, Gromacs, VMD, PyMol, AutoDock, Modeller, etc. <br>
Programming: R, C, FORTRAN, BASH, Perl, Python, Tcl, etc. <br>
Language: Mandarin Chinese (native), English (fluent), Japanese (daily communication)

---

## REVIEWER OF JOURNALS
J Mol Graph Model, J Phys Chem, J Bioinform Comput Biol, PLoS ONE, Proteins

---

## PUBLICATIONS

*Peer Reviewed Journal Articles*

1. Stender JD, Nwachukwu JC, Kastrata I, Kim Y, Strid T, Yakir M, Srinivasan S, Nowak J, Izard T, Erumbi R, Carlson KE, Katzenellenbogen JA, **Yao XQ**, Grant BJ, Leong HS, Lin CY, Frasor J, Nettles KW, Glass C. (2017) Structural and molecular mechanisms of cytokine-mediated endocrine resistance in human breast cancer cells. *Mol Cell* in press.
2. Nwachukwu JC, Srinivasan S, Bruno NE, Nowak J, Wright NJ, Minutolo F, Erumbi R, Izard T, **Yao XQ**, Grant BJ, Kojetin DJ, Elemento O, Katzenellenbogen JA, Nettles KW. (2016) Super-resolution structural analysis predicts environmental and synthetic estrogen activity. *Cell Chem Biol* in press.
3. Jariwala S, Skjærven L, **Yao XQ**, and Grant BJ. (2016) Investigating protein sequence-structure-dynamics relationships with Bio3D-web. *J Vis Exp* in press. 
4. **Yao XQ**, Skjærven L, and Grant BJ. (2016) Rapid characterization of allosteric networks with ensemble normal mode analysis. *J Phys Chem B* 120:8276-88. 
5. Skjærven L, Jariwala S, **Yao XQ**, and Grant BJ. (2016) Online interactive analysis of protein structure ensembles with Bio3D-web. *Bioinformatics* in press.
6. **Yao XQ**, Malik RU, Griggs NW, Skjaerven L, Traynor JR, Sivaramakrishnan S, and Grant BJ. (2016) Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins. *J Biol Chem* 291:4742-53. 
7. Scarabelli G, Soppina V, **Yao XQ**, Atherton J, Moores C, Verhey K, and Grant BJ. (2015) Mapping the processivity determinants of the kinesin-3 motor domain. *Biophys J* 109:1537-40.
8. Skjaerven L, **Yao XQ**, Scarabelli G, and Grant BJ (2014) Integrating protein structural dynamics and evolutionary analysis with Bio3D. *BMC Bioinformatics* 15:399.
9. **Yao XQ** and Grant BJ. (2013) Domain-opening and dynamic coupling in the α-subunit of heterotrimeric G proteins. *Biophys J* 105:L08-10.
10. **Yao XQ**, Kimura N, Murakami S, and Takada S. (2013) Drug uptake pathways of multidrug transporter AcrB studied by molecular simulations and site-directed mutagenesis experiments. *J Am Chem Soc* 135:7474-85.
11. Kenzaki H, Koga N, Hori N, Kanada R, Li WF, Okazaki K, **Yao XQ**, and Takada S. (2011) CafeMol: a coarse-grained biomolecular simulator for simulating proteins at work. *J Chem Theory Comput* 7:1979-89.
12. **Yao XQ**, Kenzaki H, Murakami S, and Takada S. (2010) Drug export and allosteric coupling in a multidrug transporter revealed by molecular simulations. *Nature Commun* 1:117.
13. Gao M, Zhu HQ, **Yao XQ**, and She ZS. (2010) Water dynamics clue to key residues in protein folding. *Biochem Biophys Res Comm* 392:95-9.
14. Gao M, **Yao XQ**, She ZS, Liu ZR, and Zhu HQ (2010) Intermediate structure and slow hydration water dynamics in protein folding process. *Acta Physico-chimica Sinica* 26:1998-2006.
15. Kang H, **Yao XQ**, She ZS, and Zhu HQ. (2009) Water-protein interplay reveals the specificity of alpha-lytic protease. *Biochem Biophys Res Comm* 385:165-9.
16. **Yao XQ** and She ZS. (2008) Key residue-dominated protein folding dynamics. *Biochem Biophys Res Comm* 373:64-8.
17. **Yao XQ**, Zhu HQ, and She ZS. (2008) A dynamic Bayesian network approach to protein secondary structure prediction. *BMC Bioinformatics* 9:49.

*Talks*

* “Dynamic coupling and activation mechanisms of heterotrimeric G proteins”, (2016) Cytoskeleton Seminar, University of Michigan
* “Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins”, (2015) Sigtrans Seminar, University of Michigan
* “G protein allosteric networks from molecular dynamics and normal mode analysis”, (2014) Sigtrans Seminar, University of Michigan
* “Probing the activation mechanism of heterotrimeric G proteins with multiscale molecular simulations”, (2014) Tools and Technology Seminar, University of Michigan
* “Nucleotide-dependent Transitions of the alpha-subunit of heterotrimeric G proteins probed with computational methods”, (2013) Sigtrans Seminar, University of Michigan
* “Drug translocation and allosteric coupling in a multidrug transporter studied by molecular simulations”, (2011) **The 17th International Biophysics Congress (IUPAC)**, Beijing, China
* “Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulations”, (2011) The 13th Annual Meeting of Chinese Life Science Association, Kyoto, Japan
* “The uptake and export of drugs in a multidrug transporter studied by coarse-grained simulations”, (2011) **The 49th Annual Meeting of the Biophysical Society of Japan**, Himeji, Japan
* “Drug export and allosteric coupling in a multidrug transporter revealed by molecular simulations”, (2011) New Era of Biosimulations with Supercomputers, Osaka, Japan
* “Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation”, (2010) **The 47th Annual Meeting of the Biophysical Society of Japan**, Tokushima, Japan
* “Rotatory mechanism of AcrB transporter studied by coarse-grained simulation”, (2009) Workshop for Next-Generation Integrated Simulation of Living Matters on Molecular Scale, Tokyo, Japan
* “Protein sequence, structure, and dynamics”, (2007) Institute for theoretical physics, Chinese Academy of Science, Beijing, China

*Posters*

* **Yao XQ**, Malik RU, Griggs NW, Skjaerven L, Traynor JR, Sivaramakrishnan S, Grant BJ. Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* Skjaerven L, Jariwala S, **Yao XQ**, Ide J, Grant BJ. The Bio3D project: interactive tools for Structural Bioinformatics. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* Skjaerven L, **Yao XQ**, Jariwala S, Grant BJ. Bio3D: interactive tools for structural bioinformatics. (2016) The 60th Annual Meeting of Biophysical Society, LA. 
* Scarabelli G, Soppina V, **Yao XQ**, Atherton J, Moores CA, Verhey KJ, Grant BJ. Mapping the processivity determinants of the kinesin-3 motor domain. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* Li H, **Yao XQ**, Grant BJ. Comparative structure dynamic analysis of G proteins. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* **Yao XQ**, Skjaerven L, Grant BJ. Characterizing nucleotide dependent allostery in G-proteins with molecular dynamics and normal mode analysis. (2015) Symposium on Computational Discovery in Complex Systems Biology, University of Michigan.
* **Yao XQ**, Skjaerven L, and Grant BJ. G-protein allosteric networks from molecular dynamics and normal mode analysis (2015) The 59th Annual Meeting of Biophysical Society, Baltimore
* **Yao XQ**, Scarabelli G, Skjaerven L, Grant BJ. The Bio3D package: new interactive tools for Structural Bioinformatics. (2014) The 58th Annual Meeting of Biophysical Society, San Francisco
* **Yao XQ**, Jariwala S, and Grant BJ. Predicting HLA-specific drug hypersensitivity with molecular docking and molecular dynamics simulation. (2014) The 58th Annual Meeting of Biophysical Society, San Francisco
* **Yao XQ** and Grant BJ. Nucleotide-dependent transitions of the alpha subunit of heterotrimeric G-proteins probed with accelerated molecular dynamics simulations. (2013) The 57th Annual Meeting of Biophysical Society, Philadelphia
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation: exploring the binding pathways of drugs. (2011) The 55th Annual Meeting of Biophysical Society, Baltimore
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulations. (2010) The 54th Annual Meeting of the Biophysical Society, San Francisco
* Kenzaki H, Koga N, Fujiwara S, Hori N, Kanada R, Okazaki K, **Yao XQ**, Li WF, Takada S. Biomolecular coarse-grained simulation program CafeMol. (2010) The 54th Annual Meeting of the Biophysical Society, San Francisco 
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of multidrug transporter studied by coarse-grained simulations. (2010) The 48th Annual Meeting of the Biophysical Society of Japan, Sendai, Japan
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation. (2010) The 2nd Biosupercomputing Symposium, Tokyo, Japan
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation. (2010) The 4th International Symposium on Molecular Science of Fluctuations toward Biological Functions, Shiga, Japan

---

References are available upon request.


