<details>
<summary>Tools</summary>

<h3>Learning R</h3>
<ul>
	<li><a href="http://thegrantlab.org/bio3d/user-guide">New to R (Grant Lab)</a></li>
	<li>Try R</li>
	<ul>
		<li>a nifty interactive R tutorial in your web browser</li>
		<li><a href="http://tryr.codeschool.com">link</a></li>
	</ul>
	<li>An introduction to R</li>
	<ul>
		<li>The official R manual</li>
		<li><a href="http://cran.r-project.org/doc/manuals/R-intro.pdf">link</a></li>
	</ul>
	<li>Learn R</li>
	<ul>
		<li>Learn by doing in your web browser</li>
		<li>Requires free registration</li>
		<li><a href="https://www.datacamp.com">link</a></li>
	</ul>
	<li>Dalgaard's Introductory Statistics with R</li>
	<li>Wickham's books</li>
	<li><a href="https//www.tidyverse.org">R for Data Science</a></li>
	<li>Google!</li>
</ul>

<h3>For Papers</h3>
<ul>
	<li>Google</li>
	<li>NCBI PubMed</li>
    <li>CiteULike</li>
    <li>PubChase</li>
</ul>

<h3>For Coding</h3>
<ul>
	<li>Github</li>
	<li>Bitbucket</li>
	<li>IFTTT (piped work flow)</li>
	<li>Galaxy (piped work flow)</li>
	<li>Taverna (piped work flow)</li>
	<li>Travis</li>
	<li>Slack</li>
    <li>Mailchimp (mailing list; not used very often)</li>
    <li>Jekyll (website building; check Wickham's R package webbook)</li>
    <li><a href="http://jmcglone.com/guides/github-pages">Github Pages</a></li>
    <li>Jupyter Notebook (Ju, Python, and R; interactive web programming and documenting)</li>
    <li><a href="https://cran.r-project.org/web/packages/httr/vignettes/api-packages.html">Web API client in R</a></li>
    <li><a href="https://zapier.com/learn/apis">Web API</a></li>
    <li>wikidot</li>
</ul>

<h3>Computational Chemistry & Bioinformatics</h3>
<ul>
	<li><a href="http://thegrantlab.pbworks.com">Grant lab wiki</a></li>
	<li><a href="https://www.biostars.org">Biostars Forum</a></li>
	<li><a href="http://www.ccl.net">CCL for computational chemistry</a></li>
	<li><a href="http://research.bmh.manchester.ac.uk/bryce/amber">Manchester's force fields</a></li>
	<li><a href="http://https://amberhub.chpc.utah.edu">Amber-hub</a></li>
	<li><a href="http://gaamp.lcrc.anl.gov">Force field generation for small molecules</a></li>
	<li><a href="http://www.ebi.ac.uk/thornton-srv/software/LIGPLOT">LIGPLOT</a></li>
	<li><a href="http://salilab.org/allosmod">Allosmod</a></li>
	<li>CHARMM-GUI</li>
	<li>MDTraj</li>
	<li><a href="http://mdn.cheme.columbia.edu">MDN (for network analysis of MD simulations)</a></li>
	<li><a href="http://biogeometry.cs.duke.edu/software/almdel/index.html">Almost-Delaunay contact calculation</a></li>
	<li>aMD reweighting</li>
	<ul>
		<li><a href="http://mccammon.ucsd.edu/computing/amdReweighting">Yinglong Miao</a></li>
		<li><a href="http://ambermd.org/tutorials/advanced/tutorial22">Amber Tutorial</a></li>
	</ul>
	<li><a href="https://westpa.github.io/westpa/index.html">WESTPA (Weighted Ensemble Simulations Toolkit)</a></li>
	<li>RedMDStream (CG simulations)</li>
	<li><a href="http://www.attract.ph.tum.de">ATTRACT (protein-protein docking)</a></li>
	<li><a href="http://3vee.molmovdb.org">3V (volume, cavity, tunnels, etc.)</a></li>
	<li><a href="http://molmovdb.org">Database of macromolecular movements</a></li>
	<li>PRODRG (to get ligand MOL2 file)</li>
	<li><a href="http://datastorm-open.github.io/visNetwork">visNetwork (visualize networks; support igraph)</a></li>
	<li>Infomap (community detection; available in igraph)</li>
	<li><a href="http://bioinf.iiit.ac.in/NAPS">NAPS (protein structure network)</a></li>
	<li><a href="http://pipe.sc.fsu.edu/cgi-bin/transcomp">TransComp (protein association rate estimation)</a></li>
	<li><a href="https://bionano.cent.uw.edu.pl/Software/SurfaceDiver">Surface Diver (mapping physicochemical features onto surfaces)</a></li>	
	<li><a href="http://www.haptimol.co.uk">HaptiMOL (Haptic interactive molecular viewing)</a></li>
	<li><a href="https://github.com/MobleyLab/alchemical-analysis">Alchemical analysis</a></li>
	<li>GPCR databases (<a href="http://www.gpcrdb.org">link 1</a>, <a href="http://gpcrconsortium.org">link 2</a>)</li>
	<li><a href="http://dca.rice.edu/portal/dca">DCA</a></li>
	<li><a href="http://pdbflex.org">PDBflex</a></li>
	<li><a href="https://modbase.compbio.ucsf.edu">Modbase (database of comparative homology)</a></li>
	<li><a href="https://cancer.sanger.ac.uk/cosmic">COSMIC (Catalogue of Somatic Mutations in Cancer)</a></li>
	<li><a href="http://www.cdc.gov">CDC (Center for Diseases Control and Prevention)</a></li>
	<li><a href="https://pdb-redo.eu">PDB-REDO (structure refinement)</a></li>
	<li><a href="http://prodata.swmed.edu/M2S">M2S (from Grishin lab; for corrected mapping of disease-related variants from OMIM and SwissVar)</a></li>
	<li><a href="http://predictioncenter.org/forcasp/viewtopic.php?f=30&t=6481&sid=689f1ef3ab1342bfb60da49127da72e7" CASP Forum (discuss contact, etc.)</a></li>
</ul>

<h3>Docking and Binding Free Energy Calculations</h3>
<ul>
	<li><a href="http://zinc15.docking.org">ZINC</a></li>
	<li><a href="http://www.alchemistry.org/wiki/Main_Page">Learn Alchemistry</a></li>
	<li><a href="http://www.alchemistry.org/wiki/Absolute_Binding_Free_Energy_-_Gromacs_2016"> Example of DDM (double decoupling method)</a></li>
	<li>T4 lysozyme L99A, a model binding site for small hydrophobic ligands</li>
	<li>CSD and GOLD (scoring functions)</li>
	<li><a href="http://escholarship.org/uc/item/6sd403pz">Dydration free energy of small molecules database by DL Mobley</a></li>
	<li>MMGB/PBSA</li>
	<ul>
		<li>Amber Manual</li>
		<li>Amber tutorial</li>
		<li>Minh's paper</li>
	</ul>
</ul>

<h3>Normal Mode Analysis</h3>
<ul>
	<li>NOMAD-Ref</li>
	<ul>
		<li>all atom Tirion's ENM</li>
		<li><a href="http://lorentz.immstr.pasteur.fr/nomad-ref.php">link</a></li>
	</ul>
	<li>Bio3D-web</li>
</ul>

<h3>Learning Gromacs</h3>
<ul>
	<li><a href="http://www.bevanlab.biochem.vt.edu/Pages/Personal/justin/gmx-tutorials/lysozyme/01_pdb2gmx.html">Beginning</a></li>
	<li><a href="http://www.bevanlab.biochem.vt.edu/Pages/Personal/justin/gmx-tutorials/complex/01_pdb2gmx.html">Ligand</a></li>
</ul>

<h3>Others</h3>
<ul>
	<li>DataCamp</li>
	<li>Kaggle (machine learning)</li>
	<li><a href="https://iris.ai">IRIS (An AI for literature search)</a></li>
	<li>FASEB IDP (individual development plans)</li>
	<li><a href="http://biorxiv.org">BioRxiv</a></li>
	<li><a href="https://www.lynda.com">Self learning</a></li>
	<li>Grammarly (grammar checking)</li>
	<li>vimeo</li>
	<li><a href="http://stke.sciencemag.org/content/sigtrans/suppl/2004/02/23/2004.218.re3.DC1/G_Protein-Cycle_STKE.swf">G protein signaling animation</a></li>
	<li><a href="http://rescuingbiomedicalresearch.org">??</a></li>
	<li>AWS (Amazon Web Services)</li>
	<li><a href="https://github.com/wrathematics/coop">coop</a></li>
	<li><a href="https://www.cellml.org">CellML (a database of cellular models)</a></li>
	<li><a href="https://www.acsreviewerlab.org">ACS Review Lab</a></li>
	<li><a href="https://www.cancer.org">American Cancer Society</a></li>
</ul>

<h3>Tips</h3>
<ul>
	<li>Git: undo last commit</li>
	<ul>
		<li>git reset --hard HEAD~1 (completely remove all changes in the last commit)</li>
		<li>git reset HEAD~1 (set HEADER 1 step back and keep changes)</li>
		<li>git reset --soft HEAD~1 (set HEADER 1 step back and keep changes and indexings)</li>
	</ul>
	<li>Set heap size up to 5120 in Crashplan</li>
	<ul>
		<li>double click home icon</li>
		<li>java mx 5120 restart</li>
	</ul>
	<li>To show installed files for a package</li>
	<ul>
		<li>rpm -ql [package]</li>
	</ul>
	<li>To mount usb HD with exfat format</li>
	<ul>
		<li>install exfat-utils and fuse-exfat</li>
		<li>check out <a href="http://docs.gz.ro/node/162">link</a></li>
		<li>mount.exfat ...</li>
	</ul>
	<li>Access $HOME placed on a server via NFS</li>
	<ul>
		<li>setsebool -P use_nfs_home_dirs 1</li>
	</ul>
	<li>Setup autostart</li>
	<ul>
		<li>sudo systemctl enable XXXd</li>
	</ul>
	<li>Use 'eog' to open images</li>
	<li>To show path of a running job</li>
	<ul>
		<li>pwdx [PID]</li>
	</ul>
	<li>Amber MD with distance restraints</li>
	<ul>
		<li>Distance restraint is NOT re-imaged!</li>
		<li>So, if restart file has separate two chains where the restraint is applied, do the re-image manually before resuming the simulation</li>
		<li>Alternatively, we should TURN OFF wrapping (iwrap=0) and write both trajectory and restart in binary (ioutfm=1, ntxo=2)</li>
	</ul>
	<li>Use photoshop to make a movie</li>
	<ul>
		<li>File > Scripts > Load Files into Stacks... (This option will add all your image in as one file with separate layers) and add all the images you want in the animation. </li>
    	<li>Open the Animation Palette under: Window > Timeline.</li>
    	<li>Create Frame Animation.</li>
    	<li>click on the Flyout Menu on top right corner of the animation (or layer) window and select (Make) Frames from Layers. NOTE: If your frames are backwards, click the bottom left icon to of the animation window get into "frame animation mode", select all the square frames then click the flyout again and reverse frames.</li>
    	<li>Adjust the timing of each frame by selecting the 0.1 sec. on the frame.</li>
    	<li>File > Export > Render Video. Among the many options you can save as a movie for Flv, IPhone, IPod, Quick Time, MPEG4 and so on. I recommend using compression type "H.264".</li>
    </ul>
    <li>In Bash (e.g., CentOS 7), sometimes `ls XXX.[a-c].YYY` does not produce what you expect. Have to spell it out like
    	`ls XXX.[abc].YYY`.</li>
</ul>

</details>
