---
layout: default
title: "Publications of Xin-Qiu Yao"
---

*Peer Reviewed Journal Articles (\*Joint-First, <sup>#</sup>Co-corresponding)*
- **Yao XQ**\*, Jariwala S\*, and Grant BJ. In silico prediction of HLA-associated drug hypersensitivity. *In preparation*.
- Li H, **Yao XQ**, and Grant BJ. Comparative structural dynamic analysis of GTPases. *To be submitted*.

    - 2018 -
1. **Yao XQ**<sup>#</sup>, Momin M, and Hamelberg D<sup>#</sup>. Elucidating allosteric communications in proteins with difference contact network analysis. *J Chem Inf Moldel* (2018) 58:1325-1330. [link](https://doi.org/10.1021/acs.jcim.8b00250)
2. Rodriguez-Bussey I, **Yao XQ**, Shouaib AD, Lopez J, and Hamelberg D. Decoding allosteric communication pathways in cyclophilin A with a comparative analysis of perturbed conformational ensembles. *J Phys Chem B* (2018) 122:6528-6535. [link](https://doi.org/10.1021/acs.jpcb.8b03824)
3. Momin MF, **Yao XQ**, Thor W, and Hamelberg D. Substrate sequence determines catalytic activities, domain-binding preferences, and allosteric mechanisms in Pin1. *J Phys Chem B* (2018) 122:6521-6527. [link](https://doi.org/10.1021/acs.jpcb.8b038192)  
4. Vu PJ\*, **Yao XQ**\*, Momin M, and Hamelberg D. Unraveling allosteric mechanisms of enzymatic catalysis with an evolutionary analysis of residue-residue contact dynamical changes. *ACS Catalysis* (2018) 8:2375-2384. [link](https://doi.org/10.1021/acscatal.7b04263)
    - 2017 -
5. **Yao XQ**\*, Cato MC\*, Labudde E, Beyett TS, Tesmer JJG, and Grant BJ. Navigating the conformational landscape of G protein-coupled receptor kinases during allosteric activation. *J Biol Chem* (2017) 292:16032-16043. [link](https://doi.org/10.1074/jbc.m117.807461)
6. Bouley R, Waldschmidt HV, Cato MC, Cannavo A, Song J, Cheung JY, **Yao XQ**, Koch WJ, Larsen SD, and Tesmer JJG. Structural determinants influencing the potency and selectivity of indazole-paroxetine hybrid G protein-coupled receptor kinase 2 inhibitors. *Mol Pharmacol* (2017) 92:707-717. [link](https://doi.org/10.1124/mol.117.110130)
7. Stender JD, Nwachukwu JC, Kastrata I, Kim Y, Strid T, Yakir M, Srinivasan S, Nowak J, Izard T, Erumbi R, Carlson KE, Katzenellenbogen JA, **Yao XQ**, Grant BJ, Leong HS, Lin CY, Frasor J, Nettles KW, Glass C. Structural and molecular mechanisms of cytokine-mediated endocrine resistance in human breast cancer cells. *Mol Cell* (2017) 65:1122-1135.e5. [link](https://doi.org/10.1016/j.molcel.2017.02.008)
8. Nwachukwu JC, Srinivasan S, Bruno NE, Nowak J, Wright NJ, Minutolo F, Rangarajan ES, Izard T, **Yao XQ**, Grant BJ, Kojetin DJ, Elemento O, Katzenellenbogen JA, Nettles KW. systems structural biology analysis of ligand effects on ERα predicts cellular response to environmental estrogens and anti-hormone therapies. *Cell Chem Biol* (2017) 24:35-45. [link](https://doi.org/10.1016/j.chembiol.2016.11.014)
9. Jariwala S, Skjærven L, **Yao XQ**, and Grant BJ. Investigating protein sequence-structure-dynamics relationships with Bio3D-web. *J Vis Exp* (2017) 125:e55640. [link](https://doi.org/10.3791/55640)
 
 *- 2016 -*
 
10. **Yao XQ**, Skjærven L, and Grant BJ. Rapid characterization of allosteric networks with ensemble normal mode analysis. *J Phys Chem B* (2016) 120:8276-8288. [link](https://doi.org/10.1021/acs.jpcb.6b01991)
11. **Yao XQ**\*, Malik RU\*, Griggs NW, Skjaerven L, Traynor JR, Sivaramakrishnan S, and Grant BJ. Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins. *J Biol Chem* (2016) 291:4742-4753. [link](https://doi.org/10.1074/jbc.M115.702605)
12. Skjærven L, Jariwala S, **Yao XQ**, and Grant BJ. Online interactive analysis of protein structure ensembles with Bio3D-web. *Bioinformatics* (2016) 32:3510-3512. [link](https://doi.org/10.1093/bioinformatics/btw482)
 
 *- 2008-2015 -*
 
13. Scarabelli G, Soppina V, **Yao XQ**, Atherton J, Moores C, Verhey K, and Grant BJ. Mapping the processivity determinants of the kinesin-3 motor domain. *Biophys J* (2015) 109:1537-1540. [link](https://doi.org/10.1016/j.bpj.2015.08.027)
14. Skjaerven L, **Yao XQ**, Scarabelli G, and Grant BJ. Integrating protein structural dynamics and evolutionary analysis with Bio3D. *BMC Bioinformatics* (2014) 15:399. [link](https://doi.org/10.1186/s12859-014-0399-6)
15. **Yao XQ** and Grant BJ. Domain-opening and dynamic coupling in the α-subunit of heterotrimeric G proteins. *Biophys J* (2013) 105:L08-L10. [link](https://doi.org/10.1016/j.bpj.2013.06.006)
16. **Yao XQ**, Kimura N, Murakami S, and Takada S. Drug uptake pathways of multidrug transporter AcrB studied by molecular simulations and site-directed mutagenesis experiments. *J Am Chem Soc* (2013) 135:7474-7485. [link](https://doi.org/10.1021/ja310548h)
17. Kenzaki H, Koga N, Hori N, Kanada R, Li WF, Okazaki K, **Yao XQ**, and Takada S. CafeMol: a coarse-grained biomolecular simulator for simulating proteins at work. *J Chem Theory Comput* (2011) 7:1979-1989. [link](https://doi.org/10.1021/ct2001045)
18. **Yao XQ**, Kenzaki H, Murakami S, and Takada S. Drug export and allosteric coupling in a multidrug transporter revealed by molecular simulations. *Nature Commun* (2010) 1:117. (The work was reported by public media: Nikkei Business Daily, Nikkan Kogyo Shimbun, and Kyoto Shimbun). [link](https://doi.org/10.1038/ncomms1116)
19. Gao M, Zhu HQ, **Yao XQ**, and She ZS. Water dynamics clue to key residues in protein folding. *Biochem Biophys Res Comm* (2010) 392:95-99. [link](https://doi.org/10.1016/j.bbrc.2010.01.003)
20. Gao M, **Yao XQ**, She ZS, Liu ZR, and Zhu HQ. Intermediate structure and slow hydration water dynamics in protein folding process. *Acta Physico-chimica Sinica* (2010) 26:1998-2006. [link](https://doi.org/10.3866/PKU.WHXB20100733)
21. Kang H, **Yao XQ**, She ZS, and Zhu HQ. Water-protein interplay reveals the specificity of alpha-lytic protease. *Biochem Biophys Res Comm* (2009) 385:165-169. [link](https://doi.org/10.1016/j.bbrc.2009.05.032)
22. **Yao XQ** and She ZS. Key residue-dominated protein folding dynamics. *Biochem Biophys Res Comm* (2008) 373:64-68. [link](https://doi.org/10.1016/j.bbrc.2008.05.179)
23. **Yao XQ**, Zhu HQ, and She ZS. A dynamic Bayesian network approach to protein secondary structure prediction. *BMC Bioinformatics* (2008) 9:49. [link](https://doi.org/10.1186/1471-2105-9-49)

*Talks*
* "Elucidating allosteric communications in proteins with difference contact network analysis", **The 256th ACS National Meeting & Exposition**, Boston (2018) *Accepted*
* "Allosteric modulation of G protein-coupled receptor kinase (GRK) activation," Sigtrans Seminar, University of Michigan (2017)
* "Dynamic coupling and activation mechanisms of heterotrimeric G proteins," (2016) Cytoskeleton Seminar, University of Michigan
* "Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins," (2015) Sigtrans Seminar, University of Michigan
* "G protein allosteric networks from molecular dynamics and normal mode analysis," (2014) Sigtrans Seminar, University of Michigan
* "Probing the activation mechanism of heterotrimeric G proteins with multiscale molecular simulations," (2014) Tools and Technology Seminar, University of Michigan
* "Nucleotide-dependent Transitions of the alpha-subunit of heterotrimeric G proteins probed with computational methods," (2013) Sigtrans Seminar, University of Michigan
* "Drug translocation and allosteric coupling in a multidrug transporter studied by molecular simulations," (2011) **The 17th International Biophysics Congress (IUPAC)**, Beijing, China
* "Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulations," (2011) The 13th Annual Meeting of Chinese Life Science Association, Kyoto, Japan
* "The uptake and export of drugs in a multidrug transporter studied by coarse-grained simulations," (2011) **The 49th Annual Meeting of the Biophysical Society of Japan**, Himeji, Japan
* "Drug export and allosteric coupling in a multidrug transporter revealed by molecular simulations," (2011) New Era of Biosimulations with Supercomputers, Osaka, Japan
* "Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation," (2010) **The 47th Annual Meeting of the Biophysical Society of Japan**, Tokushima, Japan
* "Rotatory mechanism of AcrB transporter studied by coarse-grained simulation," (2009) Workshop for Next-Generation Integrated Simulation of Living Matters on Molecular Scale, Tokyo, Japan
* "Protein sequence, structure, and dynamics," (2007) Institute for theoretical physics, Chinese Academy of Science, Beijing, China

*Posters*
* Vu PJ, **Yao XQ**, Momin M, and Hamelberg D. Evolutionarily conserved and divergent residue-residue contact dynamics provide insights into the allosteric regulation of cyclophilins. The 9th Southeast Enzyme Conference, Atlanta (2018)
* Vu PJ, **Yao XQ**, Momin M, and Hamelberg D. Evolutionarily conserved and divergent residue-residue contact dynamics provide insights into the allosteric regulation of cyclophilins. The 255th ACS National Meeting & Exposition, New Orleans (2018)
* Vu PJ, **Yao XQ**, Momin M, and Hamelberg D. Evolutionarily conserved and divergent residue-residue contact dynamics provide insights into the allosteric regulation of cyclophilins. The 62nd Annual Meeting of Biophysical Society, San Francisco (2018)
* **Yao XQ**, Jariwala S, Grant BJ. In silico prediction of HLA-associated drug hypersensitivity. The 61st Annual Meeting of Biophysical Society, New Orleans (2017)
* **Yao XQ**, Malik RU, Griggs NW, Skjaerven L, Traynor JR, Sivaramakrishnan S, Grant BJ. Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* Skjaerven L, Jariwala S, **Yao XQ**, Ide J, Grant BJ. The Bio3D project: interactive tools for Structural Bioinformatics. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* Skjaerven L, **Yao XQ**, Jariwala S, Grant BJ. Bio3D: interactive tools for structural bioinformatics. (2016) The 60th Annual Meeting of Biophysical Society, LA. 
* Scarabelli G, Soppina V, **Yao XQ**, Atherton J, Moores CA, Verhey KJ, Grant BJ. Mapping the processivity determinants of the kinesin-3 motor domain. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* Li H, **Yao XQ**, Grant BJ. Comparative structure dynamic analysis of G proteins. (2016) The 60th Annual Meeting of Biophysical Society, LA.
* **Yao XQ**, Skjaerven L, Grant BJ. Characterizing nucleotide dependent allostery in G-proteins with molecular dynamics and normal mode analysis. (2015) Symposium on Computational Discovery in Complex Systems Biology, University of Michigan.
* **Yao XQ**, Skjaerven L, and Grant BJ. G-protein allosteric networks from molecular dynamics and normal mode analysis (2015) The 59th Annual Meeting of Biophysical Society, Baltimore
* **Yao XQ**, Scarabelli G, Skjaerven L, Grant BJ. The Bio3D package: new interactive tools for Structural Bioinformatics. (2014) The 58th Annual Meeting of Biophysical Society, San Francisco
* **Yao XQ**, Jariwala S, and Grant BJ. Predicting HLA-specific drug hypersensitivity with molecular docking and molecular dynamics simulation. (2014) The 58th Annual Meeting of Biophysical Society, San Francisco
* **Yao XQ** and Grant BJ. Nucleotide-dependent transitions of the alpha subunit of heterotrimeric G-proteins probed with accelerated molecular dynamics simulations. (2013) The 57th Annual Meeting of Biophysical Society, Philadelphia
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation: exploring the binding pathways of drugs. (2011) The 55th Annual Meeting of Biophysical Society, Baltimore
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulations. (2010) The 54th Annual Meeting of the Biophysical Society, San Francisco
* Kenzaki H, Koga N, Fujiwara S, Hori N, Kanada R, Okazaki K, **Yao XQ**, Li WF, Takada S. Biomolecular coarse-grained simulation program CafeMol. (2010) The 54th Annual Meeting of the Biophysical Society, San Francisco 
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of multidrug transporter studied by coarse-grained simulations. (2010) The 48th Annual Meeting of the Biophysical Society of Japan, Sendai, Japan
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation. (2010) The 2nd Biosupercomputing Symposium, Tokyo, Japan
* **Yao XQ**, Kenzaki H, and Takada S. Functionally rotating mechanism of a multidrug transporter studied by coarse-grained simulation. (2010) The 4th International Symposium on Molecular Science of Fluctuations toward Biological Functions, Shiga, Japan
